src/invariants/token.ts

```diff
@@ -1,7 +1,47 @@
 import { BigNumber } from "bignumber.js";
 import { ERC20Contract } from "../wrappers";
 
+import * as singleLineString from "single-line-string";
+
+export const TokenAssertionErrors = {
+    MISSING_ERC20_METHOD: () =>
+        singleLineString`Contract does not implement ERC20 interface.`,
```

Comment:
Let's add a parameter for `contractAddress`, i.e.

`Contract at address ${contractAddress} does not implement ERC20 interface.`